pkgs/os-specific/linux/util-linux/default.nix

```diff
@@ -57,7 +60,8 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkg-config ];
   buildInputs =
     [ zlib pam libcap_ng ]
-    ++ lib.filter (p: p != null) [ ncurses systemd ];
+    ++ lib.filter (p: p != null) [ ncurses systemd ]
+    ++ lib.optional stdenv.hostPlatform.isStatic [ audit ];
```

Comment:
Also please add a comment what requires this.

Reply:
Fixed.